servicetalk-traffic-resilience-http/src/main/java/io/servicetalk/traffic/resilience/http/TrafficResilienceHttpClientFilter.java

```diff
@@ -401,11 +401,12 @@ public Builder circuitBreakerPartitions(
          * Requests that fail due to capacity limitation, are good candidates for a retry, since we anticipate they are
          * safe to be executed again (no previous invocation actually started) and because this maximizes the success
          * chances.
-         * @param policy The {@link PeerRejectionPolicy} that represents the peer capacity rejection behavior.
+         * @param policy The {@link ClientPeerRejectionPolicy} that represents the peer capacity rejection behavior.
          * @return {@code this}.
+         * @see ClientPeerRejectionPolicy#DEFAULT_PEER_REJECTION_POLICY
          */
-        public Builder peerRejection(final PeerRejectionPolicy policy) {
-            this.peerRejectionPolicy = requireNonNull(policy);
+        public Builder peerRejection(final ClientPeerRejectionPolicy policy) {
```

Comment:
Should we make service/client builder method names consistent? Examples: `onRejectionPolicy` or `rejectionPolicy`